rest/src/main/groovy/whelk/rest/api/ESQueryLensBoost.groovy

```diff
@@ -0,0 +1,121 @@
+package whelk.rest.api
+
+import whelk.JsonLd
+
+/**
+ * A utility to compute a list of boosted fields for ElasticSearch.
+ * The fields are computed from lens definitions of chips ans cards. It is
+ * currently contingent on the JsonLd.SEARCH_KEY mechanism to avoid the
+ * repetition of nested fields.
+ */
+class ESQueryLensBoost {
+
+    JsonLd jsonld
+
+    ESQueryLensBoost(JsonLd jsonld) {
+        this.jsonld = jsonld
+    }
+
+    List<String> computeBoostFieldsFromLenses(String[] types) {
+        def boostFields = []
+        def seenKeys = [] as Set
+
+        def chipsLenses = jsonld.displayData.lensGroups?.chips
+        def cardsLenses = jsonld.displayData.lensGroups?.cards
+
+        boostFields += [ "$JsonLd.SEARCH_KEY^100" as String ]
```

Comment:
Maybe extract all boost values to constants? They are only used once, but I prefer `CHIP_BOOST_VALUE` to `200`.

Reply:
Turning boosts into constants!